client/executor/src/wasm_runtime.rs

```diff
@@ -201,16 +199,14 @@ impl RuntimeCache {
 	///
 	/// `allow_missing_func_imports` - Ignore missing function imports.
 	///
-	/// `max_runtime_instances` - The size of the instances cache.
-	///
 	/// `f` - Function to execute.
 	///
 	/// `H` - A compile-time list of host functions to expose to the runtime.
 	///
 	/// # Returns result of `f` wrapped in an additional result.
 	/// In case of failure one of two errors can be returned:
 	///
-	/// `Err::InvalidCode` is returned for runtime code issues.
+	/// `Err::RuntimeConstruction` is returned for runtime construction issues.
```

Comment:
where is this error raised from? I don't see it in this function.

Reply:
Good point, this error also doesn't seem to be used anywhere.

Reply:
#[error(transparent)]
RuntimeConstruction(#[from] WasmError),